sql/clients_monthly_v1.sql

```diff
@@ -0,0 +1,44 @@
+WITH
+  _current AS (
+  SELECT
+    client_id,
+    sample_id,
+    ARRAY(
+    SELECT
+      AS STRUCT clients_daily_v6.* EXCEPT (client_id,
+        sample_id)) AS days
+  FROM
+    clients_daily_v6
+  WHERE
+    submission_date_s3 = @submission_date ),
+  _previous AS (
+  SELECT
+    client_id,
+    sample_id,
+    ARRAY(
+    SELECT
+      day
+    FROM
+      UNNEST(days) AS day
+    WHERE
+      submission_date_s3 > DATE_SUB(@submission_date, INTERVAL 28 DAY)) AS days
+  FROM
+    clients_monthly_v1
+  WHERE
+    submission_date = DATE_SUB(@submission_date, INTERVAL 1 DAY) )
+SELECT
+  @submission_date AS submission_date,
+  CURRENT_DATETIME AS generated_time,
+  client_id,
+  sample_id,
+  ARRAY_CONCAT(_previous.days, _current.days) AS days
```

Comment:
Does the order here need to be inverted for `days[SAFE_ORDINAL(1)] AS last_seen` to work? I was expecting us to prepend the current day, so this array would be ordered with the most recent observation first in the array.

Reply:
yes, i kept thinking about this and forgetting to go fix it >_<